src/systems/filecoin_vm/interpreter/vm_registry.go

```diff
@@ -15,6 +15,7 @@ var (
 
 // CodeIDs for system actors
 var (
+	SystemActorCodeID         = actor.CodeID_Make_Builtin(actor.BuiltinActorID_System)
```

Comment:
In line with codeID checks elsewhere, you probably want to check for this code in the places where the `addr.SystemActorAddr` is currently checked. Otherwise this change has zero semantic impact.